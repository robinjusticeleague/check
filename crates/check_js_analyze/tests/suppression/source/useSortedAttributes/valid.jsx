/** should not generate diagnostics */
<>
	{/* check-ignore assist/source/useSortedAttributes: test */}
	<Hello
		lastName="Smith"
		firstName="John"
	/>;
	{/* check-ignore assist/source/useSortedAttributes: test */}
	<Hello tel={5555555} address="NY" {...this.props} lastName="Smith" firstName="John" />;
	{/* check-ignore assist/source/useSortedAttributes: test */}
	<Hello a10="" a9="" A="" ></Hello>;
</>
