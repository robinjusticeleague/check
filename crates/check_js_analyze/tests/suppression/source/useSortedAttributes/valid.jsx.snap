---
source: crates/check_js_analyze/tests/spec_tests.rs
expression: valid.jsx
---
# Input
```jsx
/** should not generate diagnostics */
<>
	{/* check-ignore assist/source/useSortedAttributes: test */}
	<Hello
		lastName="Smith"
		firstName="John"
	/>;
	{/* check-ignore assist/source/useSortedAttributes: test */}
	<Hello tel={5555555} address="NY" {...this.props} lastName="Smith" firstName="John" />;
	{/* check-ignore assist/source/useSortedAttributes: test */}
	<Hello a10="" a9="" A="" ></Hello>;
</>

```
